src/DynamoCore.net6.sln

```diff
@@ -93,6 +93,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Watch3DNodeModels", "Librar
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Migrations", "Migrations\Migrations.csproj", "{06B9E5B0-7C50-4351-9D88-E159DC25755F}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{15CCC76A-F02A-4EFA-97A2-51A1343E7457}"
```

Comment:
what is this ? an empty project ?
Is this for dotnet6 ? maybe specify core or cross platform or something in the name ?

Reply:
it's just a folder, I will remove it for now.